microcosm/tests/config/test_validation.py

```diff
@@ -247,6 +247,38 @@ def test_typed_converted(self):
             ),
         ))
 
+    @check_no_warnings()
+    def test_boolean_typed_converted(self):
```

Comment:
maybe also add some tests with true to be safe?

Reply:
Done.